plugin/mutate/README.md

```diff
@@ -69,6 +69,14 @@ Generate a database of all mutation points:
 dextool mutate analyze
 ```
 
+Create a file `build.sh` that will build the subject under test when invoked:
```

Comment:
Instead of adding a new entry change name of compile.sh further down in the readme.